ytfzf

```diff
@@ -485,33 +533,23 @@ interface_text () {
 			video_info_text
 		done |
 		column -t -s "$tab_space" |
-		fzf -m --tabstop=1 --layout=reverse |
-		trim_id > "$selected_id_file"
+		fzf -m --tabstop=1 --layout=reverse
 }
 #}}}
 
 # External interface {{{
 interface_external () {
 	video_json_file=$1
-	selected_id_file=$2
-
-	# video_info_text can be set in the conf.sh, if set it will be preferred over the default given below
-	TTY_COLS=$external_menu_len
```

Comment:
external length variable would be affected here. Maybe its better to have 2 the variables set separately for both cases rather than a function

Reply:
Oh, oops didn't realize this, we're going to keep the function since it reduces a lot of code, the pattern could be like this,

1. call the function
2. set any lengths that weren't set properly